.travis.yml

```diff
@@ -12,8 +12,8 @@ script:
   - cargo build --verbose --all --release
   - cargo test --verbose --all --release
   - cargo install --path=. --force
-  - (cd ./example_projects/simple-binary/ && bash run.sh)
-  - (cd ./example_projects/simple-library/ && bash run.sh)
+  - (cd ./example_projects/simple-binary/ && bash run.sh && cd ../../ )
+  - (cd ./example_projects/simple-library/ && bash run.sh && cd ../../ )
```

Comment:
Not necessary: the parentheses create a subshell in which the current working directory is already independent of the parent shell.

Reply:
Thats what I thought, too. But I saw some strange failures on CI. Admittedly, my shell-foo is very low!